source/releasenotes/2024-04-08-wordpress-composer-managed-update.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Updates to WordPress (Composer Managed) Upstream
+published_date: "2024-04-08"
+categories: [wordpress, action-required]
+---
+
+Today we are pushing out an update to the [WordPress (Composer Managed)](/guides/wordpress-composer/pre-ga/wordpress-composer-managed) upstream to resolve a particular log message that started appearing after our [WP-CLI wrapper update](/release-notes/2024/02/wp-cli-changes) in February.
+
+This update adds a new file to the `/config` directory, `application.pantheon.php`. In order to accomodate this new file, a line has been added to `config/application.php` to load the `config/application.pantheon.php` file. If you have made modifications to your `config/application.php`, it's possible you may run into conflicts with this update. If this is the case, you can manually resolve by running the following command:
+
+```bash
+git pull -Xtheirs https://github.com/pantheon-upstreams/wordpress-composer-managed.git main
```

Comment:
There might a chance, but it's not likely. When we first did the Bedrock upstream, we said "don't edit `application.php`". Then, we relaxed that when we realized that that's where all the wp-config stuff goes and they would probably want/need to use it for the same sort of configuration/customization/constants. But, the WordPress upstream has a file that we maintain (`wp-config-pantheon.php`) and this one doesn't and never has.  So, there's a chance folks have customized their `application.php` and we're making changes to it. But we won't be making changes in a place they're likely to have edited or in a way that would break things significantly. We're not removing anything. 

But also: this is still EA, not GA. Things can break in EA, and we've documented that the upstream is early access in multiple places.

Reply:
Got it, that all sounds reasonable. I would be tempted to update the note to include a warning of "If you did the bad thing and edited this file, you might lose changes", because giving people a heads up is rarely wasted words

Reply:
Yeah, I might add a link to the docs page on how to manage merge conflicts since I just (re)found it yesterday